### PR TITLE
Add textField for Forms

### PR DESCRIPTION
**Pull request by vrom911**
🔥 🚒 🔥 🚒 🔥 

Ctr+d now deletes the whole input in owner field.
Changed colours just to play
Ctrl+Enter doesn't work

Resolves 💢

### CODE REVIEW

File: src/Summoner/Tui.hs [hunk 1]
```diff
@@ -112,16 +113,16 @@ mkForm = setFormConcat arrangeColumns . newForm
         ]
     )
   where
-    label :: String -> Widget n -> Widget n
-    label s w = str s <+>  w
+    label :: Text -> Widget n -> Widget n
+    label l w = txt l <+>  w
```
Comment (ChShersh) [inline src/Summoner/Tui.hs]: Heh

screenshot from 2018-11-20 23-10-05

Comment (vrom911) [reply src/Summoner/Tui.hs]: Oh no 😢

**Comment by willbasky** [comment]
My general suggestion is to make parameters longer and semantic.

**Comment by vrom911** [comment]
If you're talking about the var names it's mostly because it's copy pasted from sources. But we will do cleaning later along with more docs + common functions. Just need to make the approximate picture and to prettify it later 🙂

**Comment by willbasky** [comment]
@vrom911 then approved!

**Pull request merged**